mlrun/datastore/s3.py

```diff
@@ -109,18 +109,14 @@ def get_spark_options(self):
             res["spark.hadoop.fs.s3a.aws.profile"] = st.get("profile")
         return res
 
-    def get_filesystem(self, silent=False):
+    def get_filesystem(self):
         """return fsspec file system object, if supported"""
         if self._filesystem:
             return self._filesystem
         try:
             import s3fs  # noqa
         except ImportError as exc:
-            if not silent:
-                raise ImportError(
-                    "AWS s3fs not installed, run pip install s3fs"
-                ) from exc
-            return None
+            raise ImportError("AWS s3fs not installed, run pip install s3fs") from exc
```

Comment:
```suggestion
            raise ImportError("AWS s3fs not installed") from exc
```
The user should _not_ install the latest s3fs. They should install `mlrun[complete]` or  `mlrun[s3]` with a version that is compatible with the mlrun service on their env.

Reply:
I realize it wasn't introduced in this PR, though.